packages/wallets/src/features/cfd/screens/EnterPassword/EnterPassword.tsx

```diff
@@ -34,6 +34,7 @@ const EnterPassword: React.FC<TProps> = ({
     const title = PlatformDetails[platform].title;
     const marketTypeTitle =
         platform === PlatformDetails.dxtrade.platform ? accountType : MarketTypeDetails[marketType].title;
+    const passwordErrorHints = `Hint: You may have entered your Deriv password, which is different from your ${title} password.`;
```

Comment:
why did you move this string to a variable?

Reply:
looks cleaner in my opinion and more readable.

Reply:
ok. But I am against once-used variable...

Reply:
😀